core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/resolvers/InternalLinkResolverImpl.java

```diff
@@ -103,6 +103,7 @@ public void validate(Result result, ResourceResolver resourceResolver) {
         }
         Status status = checkLink(result.getValue(), resourceResolver);
         if (status.getCode() == HttpStatus.SC_NOT_FOUND && StringUtils.isNotBlank(internalLinksHost)) {
+            result.setValue("https://" + internalLinksHost + result.getValue());
```

Comment:
Must check if `internalLinkHost` is already configured with a schema prefix, not to add `http://` for the second time in any circumstances. Also, check to avoid a double slash in between `internalLinksHost` and `result.getValue()``